plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java

```diff
@@ -37,6 +37,7 @@
 
 	int openshiftPort = 8443;
 	String openshiftHost = "https://10.1.2.2";
+	String defaultUser = "openshift-dev";
```

Comment:
same here, a constant would help.
And then I dont get it, why do we have a hard-coded default, which we then overwrite?

Reply:
@robstryker where is this being used? In my test user "developer" was used.

Reply:
openshift-dev is the default credentials for CDK.  developer is for minishift.